lib/vquic/msh3.c

```diff
@@ -529,7 +668,7 @@ static void cf_msh3_close(struct Curl_cfilter *cf, struct Curl_easy *data)
 
   (void)data;
   if(ctx) {
-    cf_msh3_ctx_clear(ctx);
+    cf_msh3_ctx_clear(cf, data);
```

Comment:
No need to check `if(ctx)` since `cf_msh3_ctx_clear` already handles that. But really, this function should simply be merged with `cf_msh3_ctx_clear`. No need to have separate ones, IMO.